designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/Button.kt

```diff
@@ -171,10 +208,10 @@ private fun SimpleButton(
         modifier = modifier,
         enabled = enabled,
         colors = buttonColors,
-        shape = ButtonDefaults.outlinedShape,
+        shape = RoundedCornerShape(Radius.Full),
```

Comment:
Why don't use our custom shapes: `Shape.Full` will match with the shape that you are creating.

Reply:
Good idea!